src/js/components/SchemaForm.js

```diff
@@ -1,24 +1,34 @@
-import {Tooltip} from 'reactjs-components';
+import mixin from 'reactjs-mixin';
 import React from 'react';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+import {Tooltip} from 'reactjs-components';
 
+import FormUtil from '../utils/FormUtil';
 import GeminiUtil from '../utils/GeminiUtil';
 import SchemaFormUtil from '../utils/SchemaFormUtil';
 import SchemaUtil from '../utils/SchemaUtil';
 import TabForm from './TabForm';
+import Util from '../utils/Util';
 
 const METHODS_TO_BIND = [
-  'validateForm', 'getTriggerTabFormSubmit', 'handleFormChange',
-  'handleExternalSubmit'
+  'getAddNewRowButton',
+  'getRemoveRowButton',
+  'getTriggerTabFormSubmit',
+  'handleFormChange',
+  'handleExternalSubmit',
+  'validateForm'
 ];
 
-class SchemaForm extends React.Component {
+class SchemaForm extends mixin(StoreMixin) {
   constructor() {
     super();
 
     METHODS_TO_BIND.forEach((method) => {
       this[method] = this[method].bind(this);
     });
 
+    this.store_listeners = [];
```

Comment:
Just so I understand, are you adding StoreMixin because you're going to use it later or because components that extend this might want to use it?